papermario/Makefile

```diff
@@ -71,34 +75,38 @@ $(BUILD_DIR):
 	mkdir $(BUILD_DIR)
 
 $(BUILD_DIR)/$(TARGET).elf: $(O_FILES) $(LD_SCRIPT) $(BUILD_DIR)
-	$(LD) $(LDFLAGS) -o $@ $(O_FILES)
+	@$(LD) $(LDFLAGS) -o $@ $(O_FILES)
 
 $(BUILD_DIR)/%.o: %.s $(BUILD_DIR)
-	$(AS) $(ASFLAGS) -o $@ $<
+	@$(AS) $(ASFLAGS) -o $@ $<
 
 $(BUILD_DIR)/%.o: %.c $(BUILD_DIR)
-	$(CC) $(CFLAGS) -o $@ $<
+	@$(CC) $(CFLAGS) -o $@ $<
 
 $(BUILD_DIR)/%.o: %.bin $(BUILD_DIR)
-	$(LD) -r -b binary -o $@ $<
+	@$(LD) -r -b binary -o $@ $<
 
-$(BUILD_DIR)/%.YAY.o: %.YAY $(BUILD_DIR)
-	$(LD) -r -b binary -o $@ $<
+$(BUILD_DIR)/%.yay0.o: %.yay0 $(BUILD_DIR)
+	@$(LD) -r -b binary -o $@ $<
 
 $(BUILD_DIR)/%.FS.o: %.FS $(BUILD_DIR)
-	$(LD) -r -b binary -o $@ $<
+	@$(LD) -r -b binary -o $@ $<
+
+$(BUILD_DIR)/%.bgm.o: %.bgm $(BUILD_DIR)
+	@$(LD) -r -b binary -o $@ $<
 
 $(BUILD_DIR)/$(TARGET).bin: $(BUILD_DIR)/$(TARGET).elf
 	$(OBJCOPY) $< $@ -O binary
 
 # final z64 updates checksum
 $(TARGET).z64: $(BUILD_DIR)/$(TARGET).bin
-	cp $< $@
+	@cp $< $@
+	$(N64CKSUM) $@
 
 fix_asm:
-	./fix_asm.py
+	@./fix_asm.py
 
 verify: $(TARGET).z64
-	md5sum -c checksum.md5
+	sha1sum -c checksum.sha1
```

Comment:
What's the reason from going from md5sum to sha1sum?

Reply:
I can't comment on it, but also could you remove the Map_Assets.FS? I think we should probably just .gitignore .FS files to make this easier going forward